README.md

```diff
@@ -2,6 +2,85 @@
 
 Bot for Scratch Addons Discord server
 
-[Cloud hosted on opeNode.io](https://www.openode.io/)
+Cloud hosted on [opeNode.io](https://www.openode.io/)
+
+## Features
+### General
+---
+### PotatoBoard
+Messages with more than 6 potato emoji reactions are shown in the #potatoboard channel.
```

Comment:
8

Reply:
I wrote 6 because [this](https://ptb.discord.com/channels/806602307750985799/938809898660155453/938818967747821609) message

maybe colaber meant 6 more potatos were needed to get the message on the board

Reply:
it was originally 6, then it was 7, then it was 8, now its 6 again and I'm not changing again 😤